### PR TITLE
EditArray Bug Fix + minor schema update

### PR DESCRIPTION
**Pull request by luhan2017**

### CODE REVIEW

File: libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Schemas/Dialogs/Microsoft.ChoiceInput.schema [hunk 1]
```diff
@@ -127,7 +127,7 @@
                 }
             }
         },
-        "cardAction": {
+        "CardAction": {
```
Comment (tomlm) [inline libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Schemas/Dialogs/Microsoft.ChoiceInput.schema]: >CardAction [](start = 9, length = 10)

no, this should be "cardAction"

**Comment by tomlm** [comment]
>                     "$ref": "#/definitions/cardAction",

it needs to match this

---
Refers to: libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Schemas/Dialogs/Microsoft.ChoiceInput.schema:116 in 64372db. [](commit_id = 64372dbabb9483f28fe136ed85ea147242608471, deletion_comment = False)

**Comment by luhan2017** [comment]
#2154

**Comment by Danieladu** [comment]
The action of remove should be only ”delete the first one which is matched” or “delete all the matched items”. The solution provided here is the first one.

**Pull request merged**